continuous_integration/travis/flake8_diff.sh

```diff
@@ -121,6 +121,7 @@ if [[ "$MODIFIED_FILES" == "no_match" ]]; then
 else
     # Conservative approach: diff without context so that code that
     # was not changed does not create failures
-    git diff --unified=0 $DIFF_RANGE -- $MODIFIED_FILES | flake8 --diff --show-source
+    git diff --unified=0 $DIFF_RANGE -- $MODIFIED_FILES | \
+        flake8 --config setup.cfg --diff --show-source
```

Comment:
I am slightly surprised that you need `--config setup.cfg`. I ran a quick check that seems to indicate that `setup.cfg` is used without using `--config setup.cfg`:

```bash
flake8 joblib/externals # no flake8 violations
cp -r joblib/externals /tmp
flake8 /tmp/externals # plenty of flake8 violations
```